.travis.yml

```diff
@@ -2,7 +2,9 @@ language: python
 python:
 - '2.7'
```

Comment:
An alternative might be to configure tox to only run the coverage tests in 2.7, but then I'm note sure if the `after_success: coveralls` would fail in 3.4 as there would be no coverage results to uploads...